### PR TITLE
Define human trust vs technical trust

### PR DESCRIPTION
**Pull request by scouten-adobe**

### CODE REVIEW

**Comment by scouten-adobe** [comment]
Closes #100.

Closes #121.

**Review by jacqueslatour — COMMENTED** [review]
Should/Can we add a reference to ToIP Trust Registry?

File: docs/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -756,32 +756,32 @@ This specification defines three distinct kinds of trust:
...
 
 ==== Governance trust
 
-*Governance trust* exists if the following requirements are satisfied:
+*Governance trust* exists if the following requirements are met to the satisfaction of the _<<_identity_assertion_consumer,identity assertion consumer>>:_
 
-* Am I confident in how the credential(s) were issued to a _<<_named_actor,named actor>>?_
+* Am I confident in how the credential(s) were issued to a _<<_named_actor,named actor>>?_ Do I understand the level of assurance offered by the credential issuer? Is it sufficient for me to accept the identity of the _<<_named_actor,named actor>>_ as valid and meaningful?
 * Am I confident in how the credential(s) were managed and controlled by the _<<_named_actor,named actor>>,_ _<<credential_holder,credential holder>>,_ and issuer, and then used to sign the content I'm consuming?
 * If the issuer is referenced on a trust list, do I trust the governance of that trust list? In other words, do I have confidence in the process by which this issuer was added to and has remained on the trust list?
```
[the remark this answers is not in the document]
Comment (scouten-adobe) [reply docs/modules/ROOT/pages/index.adoc]: Good suggestion. Added just now.

**Review by jacqueslatour — APPROVED** [review]
Looks good to me.

**Pull request merged**